tests/test__influx_reporter.py

```diff
@@ -21,20 +21,20 @@ def setUp(self):
     def tearDown(self):
         super(TestInfluxReporter, self).tearDown()
 
-    def test_report_now(self):
+    def test_not_called_on_blank(self):
```

Comment:
what about the test_report_now - you replaced it with not to call on blank but I assume having a test for report now is still something needed ?

Reply:
report now is not suppose to send anything if it doesn't have anything to send, that's the whole point. Sending valid data is covered by a lot of other tests.